src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts

```diff
@@ -89,6 +139,11 @@ export class HuiStatisticsGraphCardEditor
                     label: localize(
                       `ui.panel.lovelace.editor.card.statistics-graph.periods.${period}`
                     ),
+                    disabled:
+                      period === "5minute" &&
+                      statisticIds?.some((statistic_id) =>
+                        statistic_id.includes(":")
```

Comment:
Maybe this should be a helper, instead of hardcoded searching for `":"`?

Reply:
Yeah, I don't think anyone will understand why this is done without at least a comment